src/bindings/python/tests/test_onnx/test_ops_matmul.py

```diff
@@ -154,7 +155,6 @@ def test_gemm_transpositions(data, kwargs, description):
 
 
 def test_gemm_flatten():
-    # input_a.shape is (4,1)
```

Comment:
I have doubt about this removal...
 @jiwaszki should we keep it? Isn't it a comment about test?

Reply:
I was not sure about it, since there is not `input_a` in the test code additionally it causes ` E800 Found commented out code` error, so if you want to keep it, it has to be either ignored or changed

Reply:
Tbh I do not even see why this comment is left here. Shape can be deduced just by looking at input. As @bszmelcz mentioned it can be changes instead, what about: `#  input_a has a shape of (4, 1)` ?

Reply:
I'm ok to remove it